verification/curator-service/ui/src/components/BulkCaseForm.tsx

```diff
@@ -475,11 +482,13 @@ class BulkCaseForm extends React.Component<
                                 </Typography>
                             </div>
                             <Form>
-                                <div className={classes.formSection}>
-                                    <Source></Source>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <FileUpload></FileUpload>
+                                <div className={classes.allFormSections}>
```

Comment:
You can use `<fieldset>` here to get an outline that matches the CaseForm

Reply:
Ah, right. Done.